spark/v2.4/spark/src/test/java/org/apache/iceberg/spark/source/TestDataFrameWrites.java

```diff
@@ -283,7 +283,7 @@ private Dataset<Row> createDataset(Iterable<Record> records, Schema schema) thro
   @Test
   public void testNullableWithWriteOption() throws IOException {
     Assume.assumeTrue(
-        "Spark 3.0 rejects writing nulls to a required column", spark.version().startsWith("2"));
+        "Spark 3.x rejects writing nulls to a required column", spark.version().startsWith("2"));
```

Comment:
here and in a few other places I think it may be ok to just put in Spark 3 rather than 3.x